web/crux-ui/src/components/templates/apply-template-card.tsx

```diff
@@ -35,6 +35,7 @@ const ApplyTemplateCard = (props: ApplyTemplateCardProps) => {
     initialValues: {
       name: propsTemplate.name,
       description: propsTemplate.description,
+      technologies: propsTemplate.technologies,
```

Comment:
Why the template apply requires the technologies? I thought it's only for UI.

If the `ApplyTemplate` model is the problem, you could do something like:
```
export type ApplyTemplate = Omit<Template, 'technologies'> & {
  type: ProductType
}
```

Reply:
Yea, that was a problem 🥲 Thanks